src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,9 +3,46 @@
 public class Comment{
     private final String name;
     private final String comment;
+    private final long id;
 
-    public Comment(String name, String comment){
+    public Comment(String name, String comment, long id){
         this.name = name;
         this.comment = comment;
+        this.id = id; 
+    }
+
+    public String getDisplayName(){
+        return name; 
+    }
+
+    public String getComment(){
+        return comment; 
+    }
+
+    public long getID(){
+        return id; 
+    }
+
+    @Override
+    public boolean equals(Object o) {   
+        if (o == this) { 
+            return true; 
+        } 
+  
+        if (!(o instanceof Comment)) { 
+            return false; 
+        } 
+         
+        Comment c = (Comment) o; 
```

Comment:
typically we use this and that to make it clear. so the code would be:

Comment that = (Comment) o;
return that.getComment().equals(this.comment)
            && that.getDisplayName.equals(this.name);